.github/workflows/trivy.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     name: Trivy 
     steps:
       - name: Checkout
-        uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4
```

Comment:
@puerco @JAORMX should we pin this to `v4.1.7` as well?

Reply:
You're right, it is the same hash. I got confused by the comment being `v4` rather than `v4.1.7`.